### ISSUE TITLE
Competing risks on big dataset (Windows only)

### ISSUE THREAD

**Issue by mnwright** [kogalur/randomForestSRC#50]
I get some "unknown software exceptions" when I run the example below on Windows (32 GB Windows 10). It runs fine when I reduce to `n=33000`. It also works fine with `n=34000` on my 2017 MacBook Pro (8GB) and on a big Linux machine. 

````R

library(randomForestSRC)

n <- 34000
p <- 4

x <- replicate(p, rnorm(n))
time <- round(runif(n, 0, 100))
status <- round(runif(n , 0, 2))
dat <- data.frame(time = time, status = status, x)

rfsrc(Surv(time, status) ~ ., dat, ntree = 5, cause = 1)
````

I have _randomForestSRC_ 2.9.1 (latest from CRAN). I tried R 3.5.1 and 3.6.0 with the same result.

**Comment by kogalur** [comment]
We're assuming that when you say it works on your MacBook and Linux installations that you mean it works with the default settings:

> rfsrc(Surv(time, status) ~ ., dat, cause = 1)
                         Sample size: 34000
                    Number of events: 17078, 8500
                     Number of trees: 1000
           Forest terminal node size: 15
       Average no. of terminal nodes: 1702.694
No. of variables tried at each split: 2
              Total no. of variables: 4
       Resampling used to grow trees: swor
    Resample size used to grow trees: 21488
                            Analysis: RSF
                              Family: surv-CR
                      Splitting rule: logrankCR *random*
       Number of random split points: 10
                          Error rate: 50.33%, 50%

> 

It's not a memory limitation issue.  We saw maybe 3.2GB maximum RAM being used with the default settings above.  We haven't tested it on Windows 10, but we suspect that you have some system files corrupted on that platform.

**Comment by mnwright** [comment]
Yes: 
> rfsrc(Surv(time, status) ~ ., dat, ntree = 5, cause = 1)
>                          Sample size: 34000
>                     Number of events: 17096, 8432
>                      Number of trees: 5
>            Forest terminal node size: 15
>        Average no. of terminal nodes: 1696.2
> No. of variables tried at each split: 2
>               Total no. of variables: 4
>        Resampling used to grow trees: swor
>     Resample size used to grow trees: 21488
>                             Analysis: RSF
>                               Family: surv-CR
>                       Splitting rule: logrankCR *random*
>        Number of random split points: 10
>                           Error rate: 50.12%, 49.69%

Same results with `rfsrc.fast()`. 

> We haven't tested it on Windows 10, but we suspect that you have some system files corrupted on that platform.

I tried with two additional Windows machines I have access to (located in another institution), one running Windows Server 2008 with 32GB memory and one Windows 10 with 8 GB. Same problem on all three machines.

**Comment by kogalur** [comment]
This has been fixed in Release 2.9.2.  Thanks.
